packages/plugin-cloudflare/src/tunnel.ts

```diff
@@ -135,17 +139,22 @@ class TunnelClientInstance implements TunnelClient {
 }
 
 function processCrashed() {
```

Comment:
On line 117 (can't leave a comment there), we handle errors when cloudflare crashes. 
We have a custom error for Rosetta, but we are ignoring any other crash we just do:
`        outputDebug('Cloudflared tunnel crashed, restarting...')`
We should include the `error.message` in that output (or even the full `error` object)